unfurl/parsers/parse_youtube.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+youtube_edge = {
+    'color': {
+        'color': '#ff0000'
+    },
+    'title': 'Youtube Parsing Functions',
+    'label': 'Y'
+}
+
+
+def run(unfurl, node):
+    youtube_domains = ['youtube.com', 'youtu.be']
+    if any(youtube_domain in unfurl.find_preceding_domain(node) for youtube_domain in youtube_domains):
+        if node.key == 't':
```

Comment:
From clicking around on YT, it looks like "time_continue" is the longer version of "t", so adding a check for that here would be good.

Ex: https://www.youtube.com/watch?time_continue=423&v=5VKTaFBlMcE&feature=emb_logo